sdk/eventhub/event-hubs/src/eventPosition.ts

```diff
@@ -226,3 +82,42 @@ export function isEarliestEventPosition(eventPosition: EventPosition): boolean {
 
   return false;
 }
+
+/**
+ * Gets the `EventPosition` corresponding to the location of the the first event present in the partition.
+ * Use this position to begin receiving from the first event that was enqueued in the partition
+ * which has not expired due to the retention policy.
+ */
+export const earliestEventPosition: EventPosition = {
+  offset: -1
+}
+
+
+/**
+ * Gets the `EventPosition` corresponding to the end of the partition, where no more events are currently enqueued.
+ * Use this position to begin receiving from the next event to be enqueued in the partion after an ``EventHubConsumer``
```

Comment:
```suggestion
 * Use this position to begin receiving from the next event to be enqueued in the partion after an `EventHubConsumer`
```

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/6729/commits/22a58a29063e681d7b172a460e6b71e1f47321e6